MULTIPLATFORM.md

```diff
@@ -0,0 +1,325 @@
+# Kotlin/Native in multiplatform projects
+
+While Kotlin/Native could be used as the only Kotlin compiler in the project, it is pretty common to combine
+Kotlin/Native with other Kotlin backends, such as Kotlin/JVM (for JVM or Android targets) or Kotlin/JS
+(for web and Node.js applications). This document describes recommended approaches and the best practices for such scenarios.
+
+Kotlin as a language provides a notion of expect/actual declarations, and Gradle, as an official Kotlin build system
+augments it with the notion of multiplatform projects (aka MPP). Those two, combined together, provide a flexible
+standartized [mechanism of multiplatform development](https://kotlinlang.org/docs/reference/multiplatform.html)
+across various Kotlin flavours.
+
+Code, common amongst multiple platforms can be placed in common modules, while platform-specific code could be placed
+into platform-specific modules, and expect/actual declarations can bind them together in developer-friendly way.
+
+One can find a step-by-step tutorial of creating a Kotlin multiplatform application for Android and iOS.
+
+## Creating multiplatform Android/iOS application with Kotlin
+
+To create MPP application one has start with clear understanding which parts of an application is common for a different
+targets, and which ones are specific, and organize module structure accordingly. For shared Kotlin code the common
+ground consist of the Kotlin's standard library, which does include basic data structures and computational primitives,
+along with expect classes with platform-specific implementation. Most frequently, such code consists of GUI,
+input-output, cryptography and other APIs, available on the particular platform.
+
+In this tutorial, the multiplatform application will include three parts:
+
+ * An **Android application** represented by a separate Android Studio project written in pure Kotlin
+ * An **iOS application** represented by a separate XCode project, written in Swift and using Kotlin
```

Comment:
I don't know if the part `and using Kotlin` is 100% correct. 🤔 
I mean. It uses a Framework which is converted from Kotlin to Objective-C.
But the iOS app don't really "use kotlin" and you will never find a single Kotlin class in Xcode...

Reply:
Right, reworded it.